<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Firewall</class>
 <widget class="QWidget" name="Firewall">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>631</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Cantarell</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>\--&gt;  CYRIAX  &lt;--/</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Cantarell</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ESTADO DEL FIREWALL</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="3">
    <widget class="QTextEdit" name="textEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>72</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans Serif';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="3">
    <widget class="QTextEdit" name="textEdit_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>72</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="tabChangesFocus">
      <bool>false</bool>
     </property>
     <property name="undoRedoEnabled">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="overwriteMode">
      <bool>false</bool>
     </property>
     <property name="tabStopWidth">
      <number>80</number>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="pushButton_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>124</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>124</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Cantarell</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CONFIGURACION</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Cantarell</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>REGISTRO DEL FIREWALL</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="6">
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QComboBox" name="comboBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="font">
      <font>
       <family>Cantarell</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <widget class="QPushButton" name="pushButton_5">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <family>Cantarell</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>LIMPIAR LOG</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4" colspan="2">
    <widget class="QPushButton" name="pushButton_6">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <family>Cantarell</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>GUARDAR LOG</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>100</horstretch>
       <verstretch>30</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Cantarell</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string notr="true">ACTIVAR</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QPushButton" name="pushButton_4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>124</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>124</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Cantarell</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>AGREGAR REGLAS</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QGraphicsView" name="graphicsView">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>21</width>
       <height>21</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="backgroundBrush">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
     <property name="foregroundBrush">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>122</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>122</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Cantarell</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ABOUT CYRIAX</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>label</zorder>
  <zorder>textEdit</zorder>
  <zorder>textEdit_2</zorder>
  <zorder>label_2</zorder>
  <zorder>pushButton_6</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>comboBox</zorder>
  <zorder>plainTextEdit</zorder>
  <zorder>graphicsView</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>abrirConf()</slot>
  <slot>abrirMostrarreglas()</slot>
 </slots>
</ui>
